pkg/client/group_mgmt_client_test.go

```diff
@@ -17,11 +18,20 @@ var (
 	client *GroupMgmtClient
 )
 
-func TestNewClient(t *testing.T) {
+func checkEnvironmentVariableExists(){
+	if (os.Getenv("SDK_TARGET_HOST") == "" ||
+		os.Getenv("SDK_TARGET_USER") == "" ||
+		os.Getenv("SDK_TARGET_USER_PASSWORD") == "") {
+			fmt.Println("ERROR: Missing one of these environment variables: SDK_TARGET_HOST, SDK_TARGET_USER, SDK_TARGET_USER_PASSWORD");
+			os.Exit(1)
+		}
+}
 
+func TestNewClient(t *testing.T) {
+	checkEnvironmentVariableExists();
```

Comment:
Add this check to TestListGetOrPost() also?  There are other ways to consolidate, but probably not worth the effort.